EventPlaning.BL.Tests/EventManagerTests.cs

```diff
@@ -30,7 +30,15 @@ public void Initialize()
                 Title = "BG and VG event",
                 DateFrom = new DateTime(2018, 01, 23),
                 DateTo = new DateTime(2018, 01, 23),
-                Address = "Test g. 93-3",
+                Type = (int)Event.EventTypes.Public,
+                Address = new Address()
+                {
+                    AddressLine1 = "Ateities g. 10",
+                    City = "Vilnius",
+                    Country = "Lithuania",
+                    Province = "Vilniaus m.",
+                    PostalCode = 08303,
```

Comment:
Will this be saved as 08303 or 8303?
Either way, I noted that there is nothing to change here. E.g. UK postal codes consist of Letters and Number like `E12 9LE`. So it at least be a string.